libsql-server/src/auth/user_auth_strategies/jwt.rs

```diff
@@ -138,6 +164,23 @@ mod tests {
         (encoding_key, decoding_key)
     }
 
+    fn key_pairs(
```

Comment:
maybe rename to generate_key_pairs
<verb>_<object> naming convention is preferred for functions
key_pair() function should also be renamed
<object> only names are best left for getters

Reply:
I'm okay with renaming both, but we should keep them consistent. It would be weird to have one be `key_pair` and the other `generate_key_pairs`.